testsuite/single_node_performance.py

```diff
@@ -150,46 +150,46 @@ class RunGroupConfig:
 #    transaction_type module_working_set_size executor_type  block_size expected_tps tps
 # )
 CALIBRATION = """
-no-op	1	VM	10000	33440	39558
-no-op	1000	VM	10000	20420.9	23305
-apt-fa-transfer	1	VM	10000	25331.5	27648
-account-generation	1	VM	10000	20798.2	22905
-account-resource32-b	1	VM	10000	29796.4	32938
-modify-global-resource	1	VM	2870	2870.9	2846
-modify-global-resource	10	VM	10000	16886.3	18041
-publish-package	1	VM	149	149.5	144
-mix_publish_transfer	1	VM	2259	2259.6	2178
-batch100-transfer	1	VM	784	784.5	773
-vector-picture30k	1	VM	112	112.2	113
-vector-picture30k	20	VM	1119	1119	1145
-smart-table-picture30-k-with200-change	1	VM	21	21.8	22
-smart-table-picture30-k-with200-change	20	VM	185	185.9	190
-modify-global-resource-agg-v2	1	VM	10000	29796.4	34840
-modify-global-flag-agg-v2	1	VM	5294	5294.9	5219
-modify-global-bounded-agg-v2	1	VM	8866	8866.4	8900
-modify-global-milestone-agg-v2	1	VM	10000	24885	27233
-resource-groups-global-write-tag1-kb	1	VM	9039	9039	9168
-resource-groups-global-write-and-read-tag1-kb	1	VM	6221	6221	6293
-resource-groups-sender-write-tag1-kb	1	VM	10000	20798.2	20711
-resource-groups-sender-multi-change1-kb	1	VM	10000	16553.6	16039
-token-v1ft-mint-and-transfer	1	VM	1276	1276.2	1280
-token-v1ft-mint-and-transfer	20	VM	10000	11483.9	12096
-token-v1nft-mint-and-transfer-sequential	1	VM	812	812.9	806
-token-v1nft-mint-and-transfer-sequential	20	VM	7732	7732.8	7854
-coin-init-and-mint	1	VM	10000	27205.9	30500
-coin-init-and-mint	20	VM	10000	23591.6	23968
-fungible-asset-mint	1	VM	10000	21180.4	23240
-fungible-asset-mint	20	VM	10000	19680.6	21859
-no-op5-signers	1	VM	10000	33440	37990
-token-v2-ambassador-mint	1	VM	10000	15320.8	16797
-token-v2-ambassador-mint	20	VM	10000	15320.8	16356
-liquidity-pool-swap	1	VM	975	975.7	998
-liquidity-pool-swap	20	VM	8194	8194.6	8497
-liquidity-pool-swap-stable	1	VM	957	957.5	955
-liquidity-pool-swap-stable	20	VM	7881	7881.5	8103
-deserialize-u256	1	VM	10000	33440	37601
-no-op-fee-payer	1	VM	2216	2216.5	2158
-no-op-fee-payer	50	VM	10000	25786.3	27457
+no-op	1	VM	0.822	1.047	38275.3
```

Comment:
The comment above still says block size, expected tps, tps.
What's the new format? also how can it work without any other changes in the PR?

Reply:
it's the line 148: 
  transaction_type module_working_set_size executor_type min_ratio max_ratio median

other list is if it is calibrated from the logs from the single run

Reply:
got it, approving. For my knowledge, where are min_ration and max_ratio taken from? based on min and max of multiple runs?

Reply:
yup

Reply:
we don't use it, but we could use that to put a tighter/looser limits based on how variable each test is.